clients/pkg/promtail/server/server.go

```diff
@@ -227,7 +227,7 @@ func (s *server) ready(rw http.ResponseWriter, _ *http.Request) {
 
 	rw.WriteHeader(http.StatusOK)
 	if _, err := rw.Write(readinessProbeSuccess); err != nil {
-		level.Error(s.log).Log("msg", "error writing success message", "error", err)
+		_ = level.Error(s.log).Log("msg", "error writing success message", "error", err)
```

Comment:
we shouldn't need this everywhere, something must not be working with https://github.com/grafana/loki/blob/main/.golangci.yml#L74-L75

Reply:
Cool, didn't know about that bit @cyriltovena. The new linter definitely felt too strict and I'm not sure why it didn't enforce these exclusions :( (perhaps I made a mistake somewhere?)

We'll either have to merge this PR ro revert https://github.com/grafana/loki/pull/4028 in order to cut a new build image :(

Reply:
may be the message has changed since then and the pattern just need correction ?